build.gradle

```diff
@@ -1,34 +1,19 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    repositories {
-        jcenter()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
-        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.1'
-
-        classpath 'com.novoda:bintray-release:0.5.0'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
-}
-
-allprojects {
     ext {
         publish_version = '1.11.0'
 
         //dependencies version
-        kotlin_version = '1.1.4-3'
+        kotlin_version = '1.1.51'
```

Comment:
Since you are on top of this already, should we change naming convention on this a bit to camelCase, just to make this consistent across the board?